<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddTotal">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>121</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddISBN">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>31</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddBookName">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>61</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>图书名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddAuthor">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>91</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddNumberAll">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>121</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddLoan">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>151</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>已借：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddVote">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>211</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>投票：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddRemain">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>181</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddISBN">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>31</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddAuthor">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>91</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddRemaining">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>181</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddLoaned">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>151</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddBookName">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>61</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddVote">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>211</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
